bigbone/src/main/kotlin/social/bigbone/MastodonClient.kt

```diff
@@ -1014,25 +1014,15 @@ class MastodonClient private constructor(
             )
         }
 
-        @Throws(InstanceVersionRetrievalException::class)
-        private fun getInstanceVersionViaApi(): String {
-            return try {
-                getInstanceVersionFromApi(2)
-            } catch (e: InstanceVersionRetrievalException) {
-                getInstanceVersionFromApi(1)
-            }
-        }
-
         /**
          * Get the version string for this Mastodon instance, using a specific API version.
```

Comment:
Minor: You could update the KDoc here, as it refers to a "specific" / "specified" Mastodon version, which is no longer the case for this method.